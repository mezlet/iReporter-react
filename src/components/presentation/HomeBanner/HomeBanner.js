import React from "react";
import { Image } from "semantic-ui-react";

const HomeBanner = () => {
  const landinImage =
    "https://res.cloudinary.com/do01bfhpw/image/upload/v1556721799/landing.png";
  return (
    <div className="home-banner-container">
      <Image src={landinImage} className="landing" />

      <h3 className="tip1">Let's Make a Better Nation</h3>
      <p className="tip2">
        iReport is a platform provided to bring any form of corruption or
        situations that requires government intervention to the notice of
        appropriate authorities
      </p>
      <p className="tip3">
        Simply{" "}
        <strong>
          <a href="UI/pages/user/signup.html">Sign Up</a>
        </strong>{" "}
        to get started
      </p>
    </div>
  );
};

export default HomeBanner;
