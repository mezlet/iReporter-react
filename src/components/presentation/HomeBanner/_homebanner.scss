.landing {
  width: 70%;
  display: block;
  margin-left: auto;
  margin-right: auto;
}
.tip1 {
  font-size: 4.5rem;
  /* font-family: Roboto; */
  text-align: center;
  letter-spacing: 0.1rem;
  line-height: 1.1;
}
.tip2 {
  font-size: 1.3rem;
  text-align: center;
  width: 65%;
  margin-left: auto;
  margin-right: auto;
  line-height: 1.5;
  letter-spacing: 0.1rem;
}

.tip3 {
  text-align: center;
  margin-top: 30px;
  font-size: 1.2rem;
}
