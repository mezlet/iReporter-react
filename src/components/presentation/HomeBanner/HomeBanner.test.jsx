import React from 'react';
import { shallow } from 'enzyme';
import HomeBanner from './HomeBanner';

const wrapper = shallow(<HomeBanner />);

describe('<HomeBanner/> rendering', () => {
  it('should render without crashing', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
