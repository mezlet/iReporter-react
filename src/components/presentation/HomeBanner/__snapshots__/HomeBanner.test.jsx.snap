// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeBanner/> rendering should render without crashing 1`] = `ShallowWrapper {}`;
